moveit_planners/chomp/chomp_interface/src/chomp_interface.cpp

```diff
@@ -38,48 +38,55 @@
+
+  node_->get_parameter_or("chomp.smoothness_cost_weight", params_.smoothness_cost_weight_, 0.1);
+  node_->get_parameter_or("chomp.obstacle_cost_weight", params_.obstacle_cost_weight_, 1.0);
+  node_->get_parameter_or("chomp.learning_rate", params_.learning_rate_, 0.01);
 
-  // nh_.param("add_randomness", params_.add_randomness_, false);
-  nh_.param("smoothness_cost_velocity", params_.smoothness_cost_velocity_, 0.0);
-  nh_.param("smoothness_cost_acceleration", params_.smoothness_cost_acceleration_, 1.0);
-  nh_.param("smoothness_cost_jerk", params_.smoothness_cost_jerk_, 0.0);
-  // nh_.param("hmc_discretization", params_.hmc_discretization_, 0.01);
-  // nh_.param("hmc_stochasticity", params_.hmc_stochasticity_, 0.01);
-  // nh_.param("hmc_annealing_factor", params_.hmc_annealing_factor_, 0.99);
-  // nh_.param("use_hamiltonian_monte_carlo", params_.use_hamiltonian_monte_carlo_, false);
-  nh_.param("ridge_factor", params_.ridge_factor_, 0.0);
-  nh_.param("use_pseudo_inverse", params_.use_pseudo_inverse_, false);
-  nh_.param("pseudo_inverse_ridge_factor", params_.pseudo_inverse_ridge_factor_, 1e-4);
+  // node_->param("add_randomness", params_.add_randomness_, false);
+  node_->get_parameter_or("chomp.smoothness_cost_velocity", params_.smoothness_cost_velocity_, 0.0);
+  node_->get_parameter_or("chomp.smoothness_cost_acceleration", params_.smoothness_cost_acceleration_, 1.0);
+  node_->get_parameter_or("chomp.smoothness_cost_jerk", params_.smoothness_cost_jerk_, 0.0);
+  // node_->param("hmc_discretization", params_.hmc_discretization_, 0.01);
+  // node_->param("hmc_stochasticity", params_.hmc_stochasticity_, 0.01);
+  // node_->param("hmc_annealing_factor", params_.hmc_annealing_factor_, 0.99);
+  // node_->param("use_hamiltonian_monte_carlo", params_.use_hamiltonian_monte_carlo_, false);
+  node_->get_parameter_or("chomp.ridge_factor", params_.ridge_factor_, 0.0);
+  node_->get_parameter_or("chomp.use_pseudo_inverse", params_.use_pseudo_inverse_, false);
+  node_->get_parameter_or("chomp.pseudo_inverse_ridge_factor", params_.pseudo_inverse_ridge_factor_, 1e-4);
 
-  nh_.param("joint_update_limit", params_.joint_update_limit_, 0.1);
+  node_->get_parameter_or("chomp.joint_update_limit", params_.joint_update_limit_, 0.1);
   // TODO: remove this warning after 06/2022
-  if (!nh_.hasParam("collision_clearance") && nh_.hasParam("collision_clearence"))
-    ROS_WARN("The param 'collision_clearence' has been renamed to 'collision_clearance', please update your config!");
-  nh_.param("collision_clearance", params_.min_clearance_, 0.2);
-  nh_.param("collision_threshold", params_.collision_threshold_, 0.07);
-  // nh_.param("random_jump_amount", params_.random_jump_amount_, 1.0);
-  nh_.param("use_stochastic_descent", params_.use_stochastic_descent_, true);
+  if (!node_->has_parameter("chomp.collision_clearance") && node_->has_parameter("chomp.collision_clearence"))
```

Comment:
Since CHOMP is new in MoveIt 2, you can clean up things like this (just keep `collision_clearance`)

Reply:
Great, fixed.